src/elements/content-explorer/ItemGrid.js

```diff
@@ -0,0 +1,56 @@
+// @flow
+import React from 'react';
+import getProp from 'lodash/get';
+import AutoSizer from 'react-virtualized/dist/es/AutoSizer';
+import GridViewWrapper from '../../components/grid-view/GridViewWrapper';
+import ItemGridCell from './ItemGridCell';
+
+type Props = {
+    canDelete: boolean,
+    canDownload: boolean,
+    canPreview: boolean,
+    canRename: boolean,
+    canShare: boolean,
+    currentCollection: Collection,
+    isSmall: boolean,
+    isTouch: boolean,
+    onItemClick: Function,
```

Comment:
It looks like these types are different between components. Do they need to be specified at each level? If so, should they be shared?

Reply:
I addressed this, but not sure if I did exactly what you wanted.  I took the common props used in `ItemGrid` and `ItemGridCell`, put them in `/src/elements/content-explorer/flowTypes.js`, and then imported that type in `ItemGrid` and `ItemGridCell`.